packages/server-wallet/src/protocols/application.ts

```diff
@@ -72,10 +91,15 @@ const fundChannel = (ps: ProtocolState): ProtocolResult | false =>
   requestFundChannelIfMyTurn(ps);
 
 const signPostFundSetup = (ps: ProtocolState): ProtocolResult | false =>
-  isPrefundSetup(ps.app.supported) &&
-  isPrefundSetup(ps.app.latestSignedByMe) &&
+  myTurnToPostfund(ps) &&
```

Comment:
Capitalize "f"?

Reply:
It seems like the file follows the convention of lower case "f" in prefund and postfund naming. So I did this instead a06a80693